doc/api/fs.md

```diff
@@ -1549,6 +1549,23 @@ Synchronous version of [`fs.readFile`][]. Returns the contents of the `file`.
 If the `encoding` option is specified then this function returns a
 string. Otherwise it returns a buffer.
 
+_Note: The behavior of `fs.readFileSync()` is platform specific. As such,
+reading a directory on macOS, Linux and Windows will return an error.
+On FreeBSD, the contents of the files in directory will be returned.
```

Comment:
What does this mean exactly? Is it just all of the files concatenated together one after the other? With or without boundaries?